packages/expo-updates/.tests/e2e/__tests__/fixtures/App.js

```diff
@@ -0,0 +1,34 @@
+import { StatusBar } from 'expo-status-bar';
+import { useEffect } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+const RETRY_COUNT = 5;
+
+export default function App() {
+  useEffect(async () => {
+    for (let i = 0; i < RETRY_COUNT; i++) {
+      const response = await fetch('http://10.0.2.2:UPDATES_PORT/notify/test');
```

Comment:
If the server is not running, this will throw instead of returning a response - wrap in try-catch